### ISSUE TITLE
How to add alt text to an image

### ISSUE THREAD

**Issue by dnkrupinski** [lylo/pagecord#1176]
**Describe the bug**
If have added to pictures to an entry. 
I have added captions to each picture. 
But there is no option for alt text.

**To Reproduce**
1. Add a new entry "New Post" 
2. Click into the post field. 
3. Add an image. 
4. Add a caption. 
5. Try to add a alt text

**Expected behavior**
The should be a field for alt text or the option to use markdown syntax.

**Screenshots**

<img width="874" height="952" alt="Image" src="https://github.com/user-attachments/assets/08eab218-8617-4c28-acc9-57c45371a2de" />

**Comment by lylo** [comment]
This is currently not possible due to the underlying tech of Pagecord and the editor. I have created a PR which opens up the possibility of this being addressed, but it's beyond my control right now. 

https://github.com/rails/rails/pull/58337

I'm hopeful we can see alt text soon though. The Lexxy team (the Pagecord editor) are aware and seem keen to solve it.
